src/odemis/dataio/test/multi_tiff_test.py

```diff
@@ -0,0 +1,101 @@
+
+import datetime
+import glob
+import json
+import logging 
+import os
+import unittest
+
+import numpy
+
+from odemis import model
+from odemis.dataio import tiff
+from odemis.util.dataio import open_acquisition
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+def write_test_metadata(path: str):
+    """Write test metadata to json file.
+    Use this function to generate test metadata in the required from a directory of *.tif images.
```

Comment:
So, if I understand correctly, you'd need to call that function again every time you add new data to the test directory? How would you call it?
Could you add a comment at the top of this file saying something like:
```
This test file requires extra data. To install it do ....
If you want to extend the data, do ....
```

Reply:
I've moved the test metadata generation to a separate script, and added instructions for running it.